### ISSUE TITLE
No episode titles in list but GUIDs instead

### ISSUE THREAD

**Issue by Killom** [Sandmann79/xbmc#755]
**Addon used**
- [X] Amazon VOD (plugin.video.amazon-test)
- [ ] Browser Launcher (plugin.program.browser.launcher)

- Addon version: 1.1.0+matrix.1

**Account type**
- [ ] primevideo.com
- [X] amazon.(com/co.uk/de/jp)

**System Setup (please provide the following information):**
- Hardware: rPi 4B 8GB
- OS version: Debian Bookworm (RaspiOS)
- Kodi version number: 20.5

**Upload Logs**
Have no logs at hand right now, but some spare time - so before I forget it again, I drop this report off.

**Describe the bug**

Sometimes there are no episodes displayed when opening a series. Instead the episodes are named GUID style and cant be played.

Is this a known issue?

**Comment by Sandmann79** [comment]
Yes, it happens from time to time. Normally it should be sufficient to refresh the relevant season or series via the context menu.
